ClientRuntimes/CSharp/ClientRuntime/Properties/AssemblyInfo.cs

```diff
@@ -6,18 +6,18 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("Microsoft Rest Client Runtime")]
-[assembly: AssemblyDescription("Provides HttpClient infrastructure for clients generated by AutoRest.")]
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.9.7.4")]
-
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyTitle("Microsoft REST Client Runtime")]
+[assembly: AssemblyDescription("Provides HttpClient infrastructure AutoRest-generated clients.")]
+[assembly: AssemblyCompany("Microsoft Corporation")]
 [assembly: AssemblyProduct("Microsoft AutoRest")]
-[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation")]
+[assembly: AssemblyCopyright("Copyright � Microsoft Corporation")]
+[assembly: AssemblyConfiguration("")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
-
-[assembly: ComVisible(false)]
 [assembly: CLSCompliant(false)]
```

Comment:
It is not related with this PR. Since we are at it, do you know why we mark it at the assembly level that we are not comply with the CLS? For client applications, it doesn't matter, but it is the common library, and would be nice that our library could be used for other .net languages. Or at least we know which type violates it and we comment it here.